lib/mail_room/delivery/postback.rb

```diff
@@ -33,6 +33,7 @@ def deliver(message)
           # request.headers['Content-Type'] = 'text/plain'
         end
 
+        MailRoom.structured_logger.info({ delivery_method: 'Postback', action: 'message pushed', url: @delivery_options.delivery_url })
```

Comment:
Each delivery method may need to grab the any logger out of the mailbox on initialize.

Reply:
That's how I did it. Let me know your thoughts. 👍